datafusion/core/src/physical_plan/planner.rs

```diff
@@ -1293,15 +1293,15 @@ impl DefaultPhysicalPlanner {
 }
 
 /// Expand and align  a GROUPING SET expression.
```

Comment:
nit (but isn't introduced in this PR):
```suggestion
/// Expand and align a GROUPING SET expression.
```